go/common/batches.go

```diff
@@ -19,9 +19,9 @@ type ExtBatch struct {
 // Hash returns the keccak256 hash of the batch's header.
 // The hash is computed on the first call and cached thereafter.
 func (b *ExtBatch) Hash() L2BatchHash {
-	if hash := b.hash.Load(); hash != nil {
-		return hash.(L2BatchHash)
-	}
+	//if hash := b.hash.Load(); hash != nil {
```

Comment:
raised a ticket to address this issue.
This was commented out for rollups as it introduced bugs